src/agat/agat_convertspgff2gtf/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
+test_dir="${meta_resources_dir}/test_data"
+
+
+echo "> Run agat_convertspgff2gtf with test data"
+"$meta_executable" \
+  --gff "$test_dir/0_test.gff" \
+  --output "output.gtf" 
+
+echo ">> Checking output"
+[ ! -f "output.gtf" ] && echo "Output file output.gtf does not exist" && exit 1
+
+echo ">> Check if output is empty"
+[ ! -s "output.gtf" ] && echo "Output file output.gtf is empty" && exit 1
+
```

Comment:
Can you also check the output's contents based on the input gff you provided?

Reply:
I added a test that extract the ID value from the first line of the GFF and then assess if the converted GTF first line contains the expected gene_id and ID. Let me know if this is not sufficient